libs/ledger-live-common/src/families/casper/deviceTransactionConfig.ts

```diff
@@ -31,14 +20,14 @@ function getDeviceTransactionConfig({
 }): Array<DeviceTransactionField> {
   const fields: Array<DeviceTransactionField> = [];
   fields.push({
-    type: "casper.amount",
+    type: "casper.extendedAmount",
     label: "Amount",
-    value: `${transaction.amount.toNumber().toLocaleString()} motes`,
+    value: transaction.amount.toString(),
```

Comment:
We should use "toFixed" as "toString" has some issues when using long long numbers.

Reply:
Ohh, that's insightful. Done!